node/src/main/scala/com/wavesplatform/network/PeerDatabase.scala

```diff
@@ -49,6 +51,8 @@ object PeerDatabase extends ScorexLogging {
 
     override def randomPeer(excluded: Set[InetSocketAddress]): Option[InetSocketAddress] = None
```

Comment:
Better remove `randomPeer` method completely and use the `randomPeers(1, ...)` as a replacement

Reply:
Done